# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../openoffice-org/t2-distro.patch
# Copyright (C) 2004 - 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

We really should send this upstream ASAP.

  - Rene Rebe <rene@exactcode.de>

--- ooo-build-2.2.1/patches/src680/apply.vanilla	2007-07-31 21:17:55.000000000 +0000
+++ ooo-build-2.2.1/patches/src680/apply	2007-07-31 21:18:52.000000000 +0000
@@ -38,6 +38,10 @@
 SUSE      : NovellBase, BerkeleyDB43
 SUSE-PPC  : NovellBase, BerkeleyDB43, SystemJavaBits
 SUSE-64   : NovellBase, BerkeleyDB43, 64bit
+# T2 SDE Project - http://t2-project.org
+T2Base : LinuxCommon, NotDebian, T2Only
+T2 : T2Base, BerkeleyDB43
+T264 : T2Bask, BerkeleyDB43, 64bit
 # Debian
 DebianBase : LinuxCommon, DebianBaseOnly, SystemJavaBits, Lockdown, STLport5, CalcSolver, CXXHelpLinker, BerkeleyDB43
 Debian : DebianBase, DebianOnly, DebianSidOnly, Debian32Only, CairoFonts
@@ -659,6 +663,9 @@
 # Some Novell colors ...
 novell-palette.diff, michael
 
+[ T2Only ]
+
+# let's see what the future brings, ...
 
 [ NotDebian ]
 
